CHANGELOG.MD

```diff
@@ -1,3 +1,22 @@
+## 1.289.1 (2024-04-17)
+
+### Features
+
+* Update golang.org/x/sys version to 0.1.0.
+* Update gopkg.in/yaml.v3 version to 3.0.0.
+* Update go version to 1.17
+* Update google.golang.org/protobuf version to 1.33.0.
+
+## 1.279.1 (2023-12-21)
+
+### Features
+
+* Improve reading config file
+* Split large spans into multiple export messages
+* Update semantic constants
```

Comment:
This & the next 2 points don't really mean anything to users, most of them are internal improvements for compatibility with future server-side features.

Reply:
Shouldn't we also put changes useful for developers only

Reply:
These are only useful for developers at Dynatrace, not outside developers, or do I misunderstand something?

Reply:
Yes, but I don't think we'll have many (if any) external contributors here.